### ISSUE TITLE
Email: Intopia Feedback - WCAG 3.0 FPWD (1 - Requirements)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#466]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0015.html): 

Accessibility Guidelines Working Group,

On behalf of Intopia, we congratulate you on the release of the first working draft of WCAG 3.0. We wanted to acknowledge this significant milestone and commend you on your work to date. These guidelines will ultimately be instrumental in increasing accessibility and inclusion.

We have reviewed the working draft and are providing feedback by e-mail to best address the questions raised in the FPWD blog post<https://www.w3.org/blog/2021/01/wcag-3-fpwd/>.

Are there additional Design Principles and Requirements that should be included in the WCAG 3 project?

The Design Principles seem comprehensive and useful. However, in comparison to similar work by other organisations, they are verbose and require considerable concentration to understand. Usually Design Principles are intended to be concise phrases which help people make decisions between choices. Having complex paragraphs which could be interpreted in different ways is not as helpful in those situations. This is also reflected in the first two items under the heading "Accessibility guidelines should", which seem to be very similar. We suggest:

*   Editing the Design Principles to be concise. This might mean a pithy phrase followed by a complete sentence or two to expand or explain the reasoning, similar to the building design principles by Design Council<https://principles.design/examples/the-principles-of-inclusive-design>, Australian Government Digital Service Standards<https://www.dta.gov.au/help-and-advice/digital-service-standard/digital-service-standard-criteria> or  Jakob Nielsen's Usability Heuristics<https://principles.design/examples/10-usability-heuristics-for-user-interface-design>
*   Clarifying the distinction between the first two Accessibility principles or merging them into one

The Requirements are also comprehensive and useful. Motivation is particularly good as a requirement – but is there a write-up of how a scoring system became the preferred way to achieve that goal? If not, it might be useful to have one to refer to as the standard moves through the draft stages. Or if the scoring system isn't seen as the only way to motivate people, maybe the reference to it needs to be removed from this Requirement.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by jspellman** [comment]
Draft Response: We have changed the Requirement for Motivation to remove a reference to a scoring system: "The Guidelines motivate organizations to go beyond minimal accessibility requirements by structuring WCAG 3 to encourage organizations which demonstrate a greater effort to improve accessibility."

**Comment by DAIDAOXING** [comment]
> [Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0015.html):
> 
> Accessibility Guidelines Working Group,
> 
> On behalf of Intopia, we congratulate you on the release of the first working draft of WCAG 3.0. We wanted to acknowledge this significant milestone and commend you on your work to date. These guidelines will ultimately be instrumental in increasing accessibility and inclusion.
> 
> We have reviewed the working draft and are providing feedback by e-mail to best address the questions raised in the FPWD blog posthttps://www.w3.org/blog/2021/01/wcag-3-fpwd/.
> 
> Are there additional Design Principles and Requirements that should be included in the WCAG 3 project?
> 
> The Design Principles seem comprehensive and useful. However, in comparison to similar work by other organisations, they are verbose and require considerable concentration to understand. Usually Design Principles are intended to be concise phrases which help people make decisions between choices. Having complex paragraphs which could be interpreted in different ways is not as helpful in those situations. This is also reflected in the first two items under the heading "Accessibility guidelines should", which seem to be very similar. We suggest:
> 
> * Editing the Design Principles to be concise. This might mean a pithy phrase followed by a complete sentence or two to expand or explain the reasoning, similar to the building design principles by Design Councilhttps://principles.design/examples/the-principles-of-inclusive-design, Australian Government Digital Service Standardshttps://www.dta.gov.au/help-and-advice/digital-service-standard/digital-service-standard-criteria or  Jakob Nielsen's Usability Heuristicshttps://principles.design/examples/10-usability-heuristics-for-user-interface-design
> * Clarifying the distinction between the first two Accessibility principles or merging them into one
> 
> The Requirements are also comprehensive and useful. Motivation is particularly good as a requirement – but is there a write-up of how a scoring system became the preferred way to achieve that goal? If not, it might be useful to have one to refer to as the standard moves through the draft stages. Or if the scoring system isn't seen as the only way to motivate people, maybe the reference to it needs to be removed from this Requirement.

**Comment by DAIDAOXING** [comment]
TODO DIRECTO A CUENTA REAL DE DAI ACTIVACION UNICO SOLO DAOXING. TARJETA MASTE CARD ♦️♦️♦️
5320 1743 0211 9495 
O
TARJETA MASTER CARD
4188 2021 1688 8882
UNICO CORREOS ELECTRONICOS DE PRESIDENT CEO TAN BIEN DIRECTOR DE PAYMENT PARA GESTIONAR TODO REDES SOCIALES : DAOXINGDAIUNICO@GMAIL.COM
NOMBRE:DAI DAOXING
EMPRESA PRINCIPAL:DAI GLOBAL SA
EMPRESA EN REDES SOCIALES:DAI GLOBAL BUSINESS SA
EMPRESA PARA GESTIONAR UNICO :DAI GLOBAL BANK DIGITAL 
NIE:5800505C
PASAPORTE DE CHINA :E44167604
UNICO PERSONAL EN TODAS LAS EMPRESAS DE REDES SOCIALES PARA GESTIONAR TODO LOS TRABSJOS 
Welcome to Gboard clipboard, any text you copy will be saved here.

**Comment by rachaelbradley** [comment]
Note that the other PR we are looking at changes this to say: The Guidelines motivate organizations to go beyond minimal accessibility requirements by providing a conformance model that rewards organizations which demonstrate a greater effort to improve accessibility.

**Comment by alastc** [comment]
I'm closing with a 'catchup' PR in #736 which uses what we agreed at https://www.w3.org/2024/02/27-ag-minutes#item05
